doc/build/html/_sources/rtd_pages/archive_git/authors.rst.txt

```diff
@@ -0,0 +1,35 @@
+=======
+Authors
+=======
+
+Zcash Contributors
+==================
+
+Jack Grigg (601) Simon Liu (297) Sean Bowe (193) Daira Hopwood (102)
```

Comment:
This file is a) out-of-date, and b) broken compared to the original. Additionally, the script that generates the file (`./zcutil/release-notes.py`) has not been updated for the new format and location.